server/lib/src/idm/server.rs

```diff
@@ -2095,6 +2119,11 @@ impl<'a> IdmServerProxyWriteTransaction<'a> {
 
     #[instrument(level = "debug", skip_all)]
     pub fn commit(mut self) -> Result<(), OperationError> {
+        if self.qs_write.get_changed_app() {
+            self.qs_write
+                .get_apprs_set()
```

Comment:
get_apprs_set probably needs a more descriptive name. I think you copy-pasted oauth2rs here, and may not have realised that the "rs" stands for "resource server". So here you probably don't want "apprs", you want something descriptive like "get_ldap_application_set".

Reply:
All names corrected and without abbreviations.